robot/src/main/java/org/frcteam2910/common/robot/drivers/Limelight.java

```diff
@@ -44,26 +68,50 @@ public Limelight(NetworkTable table) {
         snapshot = table.getEntry("snapshot");
     }
 
+    /*
+     * Returns true if the Limelight has a target, else false
```

Comment:
Start with `Checks` instead of `Returns true` and remove the stuff after the comma

Reply:
EdoN